bubuku/controller.py

```diff
@@ -61,13 +59,10 @@ def add_check(self, check):
 
     def _register_running_changes(self, ip: str) -> dict:
         _LOG.debug('Taking lock for processing')
-        with self.zk.take_lock('/bubuku/global_lock', ip):
+        with self.zk.lock(ip):
```

Comment:
On first stage it's better not to use any timeouts...
And if we can not lock, than it makes no sence to proceed with loop